examples/shouldi/.gitignore

```diff
@@ -19,5 +19,6 @@ wheelhouse/
 *.db
 htmlcov/
 tests/golangci-lint-download/
+tests/javascript_algo-download/
```

Comment:
Don't forget to add 

```
tests/npm-audit-download/
```

Reply:
oops ... missed it . Thanks @pdxjohnny .. Did it :)